src/styles/settings/star.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__stars &__star:nth-child(1n) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__stars &__star:nth-child(1n + 5) {
+    background-image: url(../images/star.svg);
+  }
+
+  &__rev {
+    display: flex-end;
```

Comment:
There is an issue with the `display: flex-end;` property. `flex-end` is not a valid value for the `display` property. If you want to align items to the end, consider using `justify-content: flex-end;` or `align-items: flex-end;` within a `display: flex;` container.